core/src/com/threecubed/auber/entities/PowerUp.java

```diff
@@ -0,0 +1,133 @@
+package com.threecubed.auber.entities;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.math.Vector2;
+import com.threecubed.auber.World;
+
+/**
+ * The power up entity that applies buffs to the player. Activated in @link "Player.java"
+ *
+ * */
+public class PowerUp extends GameEntity {
+
+    private String ability;
```

Comment:
Any reason this is a string, not an `Ability` enum?

Reply:
It gets the texture from the atlas using the name which needs to be a string, easy enough to change though

Reply:
Changed, ability now takes type Abilities rather than String